lib/src/tools/format_tool.dart

```diff
@@ -68,6 +68,11 @@ class FormatTool extends DevTool {
   /// Run `dartfmt -h -v` to see all available args.
   List<String> formatterArgs;
 
+  /// If the formatter should also organize imports.
+  ///
+  /// By default, this is disabled.
+  bool organizeImports = false;
```

Comment:
I like the name `organizeImports` 👍 It would be nice to name everything consistently, though. Could we rename `ImportCleanerTool` to `ImportOrganizerTool`?

Reply:
@evanweible-wf now that this organizes imports and exports, should I just call this `organizeDirectives`?

Reply:
Ah yeah, good call - I like `organizeDirectives` 👍